### ISSUE TITLE
Can't start LMMS without root privileges

### ISSUE THREAD

**Issue by multim0** [LMMS/lmms#4164]
When I try to start lmms using common user it refuses to start. It didn't happen while I used `--allowroot`
Log:
`
..././lmms-1.2.0-rc5-linux-x86_64.AppImage 
Carla does not appear to be installed.  That's OK, please ignore any related library errors.
/tmp/.mount_lmms-1ptH48i/AppRun: linia 17: ldconfig: nie znaleziono polecenia
Jack does not appear to be installed.  That's OK, we'll use a dummy version instead.
VST sync support disabled in your configuration
Cannot load library /tmp/.mount_lmms-1ptH48i/usr/lib/lmms/libcarlabase.so: (libcarla_standalone2.so: nie można otworzyć pliku obiektu dzielonego: Nie ma takiego pliku ani katalogu)
Cannot load library /tmp/.mount_lmms-1ptH48i/usr/lib/lmms/libcarlapatchbay.so: (libcarla_standalone2.so: nie można otworzyć pliku obiektu dzielonego: Nie ma takiego pliku ani katalogu)
Cannot load library /tmp/.mount_lmms-1ptH48i/usr/lib/lmms/libcarlarack.so: (libcarla_standalone2.so: nie można otworzyć pliku obiektu dzielonego: Nie ma takiego pliku ani katalogu)
"Cannot load library /usr/lib/ladspa/ladspa_guitarix.so: (/usr/lib/ladspa/ladspa_guitarix.so: undefined symbol: jack_midi_event_get)"
/tmp/.mount_lmms-1ptH48i/AppRun: linia 23:  3278 Naruszenie ochrony pamięci   QT_X11_NO_NATIVE_MENUBAR=1 $DIR/usr/bin/lmms.real "$@"
`

**Comment by tresf** [comment]
Are you on a platform with `locate` command?

```bash
sudo updatedb
locate .lmmsrc.xml
```
The `.lmmsrc.xml` is responsible for loading up your last settings and you may have hit a bug.

The reason we don't encourage running as root is because this file can get [locked out](https://user-images.githubusercontent.com/6345473/36064808-d240d16a-0e5e-11e8-8adf-5e5c2515e4bf.png) if `--allowroot` is provided.

e.g. 
```
ls -al /home/owner/.lmmsrc.xml 
-rw-r--r-- 1 root root 1561 Feb 10 12:33 /home/tresf/.lmmsrc.xml
```

Anyway, can you move `~/.lmmsrc.xml` to a safe place and try again?  This fixes the most common problems.  Why it's working as root is a bit of mystery.

Note, if you're logging in as a full root users, you'll have [an additional `.lmmsrc.xml` in your root's home directory](https://user-images.githubusercontent.com/6345473/36064835-48fbcd6e-0e5f-11e8-9559-76a4b27943d2.png).  The one we want a copy of is the one crashing the software from your user directory, assuming that's the culprit.

Also, can you do `export LC_ALL=C` before running to make the error messages in English?  Thanks!

**Comment by multim0** [comment]
I moved that file to another place, but that didn't resolve the problem. What's more log message is longer than before:
`Programy/./lmms-1.2.0-rc5-linux-x86_64.AppImage 
Carla does not appear to be installed.  That's OK, please ignore any related library errors.
/tmp/.mount_lmms-1QkpiCj/AppRun: line 17: ldconfig: command not found
Jack does not appear to be installed.  That's OK, we'll use a dummy version instead.
VST sync support disabled in your configuration
Cannot load library /tmp/.mount_lmms-1QkpiCj/usr/lib/lmms/libcarlapatchbay.so: (libcarla_standalone2.so: cannot open shared object file: No such file or directory)
Cannot load library /tmp/.mount_lmms-1QkpiCj/usr/lib/lmms/libcarlarack.so: (libcarla_standalone2.so: cannot open shared object file: No such file or directory)
Cannot load library /tmp/.mount_lmms-1QkpiCj/usr/lib/lmms/libcarlabase.so: (libcarla_standalone2.so: cannot open shared object file: No such file or directory)
"Cannot load library /usr/lib/ladspa/ladspa_guitarix.so: (/usr/lib/ladspa/ladspa_guitarix.so: undefined symbol: jack_midi_event_get)"
ALSA lib conf.c:3523:(snd_config_hooks_call) Cannot open shared library (null)
ALSA lib conf.c:3986:(snd_config_update_r) hooks failed, removing configuration
cannot open sequencer: No such file or directory
cannot open MIDI-device: Inappropriate ioctl for device
Couldn't create MIDI-client, neither with ALSA nor with OSS. Will use dummy-MIDI-client.
ALSA lib pcm_dsnoop.c:618:(snd_pcm_dsnoop_open) unable to open slave
ALSA lib pcm_dmix.c:1052:(snd_pcm_dmix_open) unable to open slave
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2495:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm_dmix.c:1052:(snd_pcm_dmix_open) unable to open slave
/tmp/.mount_lmms-1QkpiCj/usr/bin/lmms.real: symbol lookup error: /tmp/.mount_lmms-1QkpiCj/usr/lib/libportaudio.so.2: undefined symbol: jack_client_open
`

**Comment by tresf** [comment]
@multim0 this sounds like a duplicate of #4094.  Can you confirm that this statement from #4094 is true for your machine?

>  Interim AppImage versions before this, such as [lmms-1.2.0-rc4.29](https://github.com/tresf/lmms/releases/download/v1.2.0-rc4/lmms-1.2.0-rc4.29-linux-x86_64.AppImage) didn't suffer this problem.

In regards to how launching as root fixes it, I'm not sure, but I'll make mental note.

**Comment by multim0** [comment]
Yes, it's the same when I downloaded older version, it works fine.
